Code/src/main/java/nl/utwente/group10/ui/menu/FunctionMenu.java

```diff
@@ -114,8 +119,6 @@ protected void updateItem(FunctionEntry item, boolean empty) {
         graphBlockButton.setOnAction(event -> addBlock(new GraphBlock(parent)));
 
         // TODO remove once debugging is done
-        Button debugButton = new Button("Error All");
-        debugButton.setOnAction(event -> parent.errorAll());
```

Comment:
We are done with this part of debugging, as a Block error state is not used anymore. The argument error state can easily be tested by using (+) and (==)